src/API.jl

```diff
@@ -77,6 +100,7 @@ description(::E) where {E<:Event} = description(E)
 Add a single timestampted event into the tracefile.
 """
 function emit(::Event{T,V}; counters::Bool=false) where {T,V}
+    println("Event emit: $(T): $(V)")
```

Comment:
I don't think this is a good idea... Maybe you should replace it with `Logging.@debug`

Reply:
Yep, looks like a useful function.